string[] keys_down
string[] keys_up
float64 mouse_move_x
float64 mouse_move_y
uint32[] mouse_click
uint32[] mouse_release